packages/react-native/ReactCommon/react/renderer/graphics/Transform.cpp

```diff
@@ -262,6 +279,14 @@ bool Transform::operator==(const Transform& rhs) const {
       return false;
     }
   }
+  if (this->operations.size() != rhs.operations.size()) {
```

Comment:
nit: Since the operations and matrix are the only members, and we aren't doing any sort of clever comparison (e.g. fp error or NaN insensitive), I think we could just default this operator. vector and array already generate equality operators, so I'm not sure why the existing code was looping to do this.